gpii/node_modules/testing/src/Mocks.js

```diff
@@ -351,6 +364,15 @@ gpii.test.integration.mockSettingsHandlerRegistry.populateOne = function (that,
     fluid.setGlobalValue(that.options.rootPath + "." + key + ".get", mock.get);
 };
 
+gpii.test.integration.mockSettingsHandlerRegistry.depopulate = function (that) {
+    fluid.each(that.settingsHandlers, function (settingsHandler) {
+        if (settingsHandler.destroy) {
```

Comment:
I can see from the coverage report that this check is necessary, but I'm curious about when a settingsHandler might not have a `destroy` invoker, and wondering if this can be expressed more clearly, for example, by using `fluid.isComponent(settingsHandler)`.

Reply:
Good suggestion, adopted